src/cascadia/inc/cppwinrt_utils.h

```diff
@@ -110,17 +110,20 @@ private:                                              \
 // Use this macro to quickly implement both the getter and setter for an observable property.
 // This is similar to the GETSET_PROPERTY macro above, except this will also raise a
 // PropertyChanged event with the name of the property that has changed inside of the settter.
-#define OBSERVABLE_GETSET_PROPERTY(type, name, event)                       \
-public:                                                                            \
-    type name() { return _##name; };                                               \
-    void name(const type& value)                                                   \
-    {                                                                              \
-        _##name = value;                                                           \
-        event(*this, Windows::UI::Xaml::Data::PropertyChangedEventArgs{ L#name }); \
-    };                                                                             \
-                                                                                   \
-private:                                                                           \
-    type _##name;
+#define OBSERVABLE_GETSET_PROPERTY(type, name, event)                                  \
+public:                                                                                \
+    type name() { return _##name; };                                                   \
+                                                                                       \
+private:                                                                               \
+    const type _##name;                                                                \
+    void name(const type& value)                                                       \

```

Comment:
Wait why'd the setter implementation here become private?

Reply:
This is in reference to Dustin's comment above about how the property shouldn't be settable externally. I guess also in `Tab`'s case, anyone trying to set the Title or IconPath should really be using `SetTabText` or `UpdateIcon`, since that also updates the `TabViewItem`'s TItle and Icon.